MultiShotCallerWithLights.ino

```diff
@@ -139,50 +85,116 @@ bool sampleAndCheckIfAboveThreshold(int inputPin, int threshold, int sampleCount
   for (int i = 0; i < sampleCount; i++){
     hitSampleSum = hitSampleSum + analogRead(inputPin);
   }
+  
+  if(DEBUG){
+    Serial.print("Hit reading Average: ");
+    Serial.println((hitSampleSum/sampleCount));
+  }
 
   // Check if average is above threshold
-  if ( (hitSampleSum/sampleCount) > threshold) {
+  if ( (hitSampleSum/sampleCount) > threshold) {    
     return true;
   }
 
   return false;
 }
 
-void setHitFlash(){
-  for(int i = 0; i < NUM_LEDS; i++){
-    leds[i] = CRGB::Red;
+void clearLeds(){
+  setLEDs("black", 0, NUM_LEDS, NULL);
+}
+
+void setLEDs(String color, int startLED, int endLED, int brightLevel){
+  int hue = 0;
+  int saturation = 255;
+  int brightness = LED_BRIGHTNESS;
+  if(brightLevel){
+    brightness = brightLevel;
+  }
+
+  if(color.equals("red")){
+    hue = 0;
+  } else if(color.equals("orange")){
+    hue = 22;
+  } else if(color.equals("green")){
+    hue = 85;
+  } else if(color.equals("black")){
+    brightness = 0;
+  }
+  
+  for(int i = startLED; i < endLED; i++){
+    leds[i].setHSV( hue, saturation, brightness);
```

Comment:
I believe you can do `FastLed.setBrightness(value)` to set the brightness across the board without having to do all this extra brightness logic logic FYI